src/store/reducers/tezos/blokchain-blocks.ts

```diff
@@ -1,26 +0,0 @@
-import {
-  BLOKCHAIN_SET_TRANSACTIONS,
-  BLOKCHAIN_FLUSH_TRANSACTIONS,
-  BLOKCHAIN_SET_MORE_TRANSACTIONS,
-  BLOKCHAIN_FILTER_TRANSACTIONS
-} from '../../actions/tezos/blokchain';
-import { Block } from '../../../types';
-
-const initState: Block[] = [];
-
-const blocks = (state = initState, action: any): Block[] => {
-  switch (action.type) {
-    case BLOKCHAIN_FLUSH_TRANSACTIONS:
-      return initState;
-    case BLOKCHAIN_SET_TRANSACTIONS:
```

Comment:
I would change `BLOKCHAIN` to `TEZOS` and keep it, we will be using it in the next sprint. Same situation as for ethereum with infinity scroll for Milena's list of transactions

Reply:
done